ui/app/src/views/import/GrafanaFlow.tsx

```diff
@@ -48,11 +53,24 @@ function GrafanaFlow({ dashboard }: GrafanaFlowProps) {
     navigate(`/projects/${data.metadata.project}/dashboards/${data.metadata.name}`);
   });
 
+  // initialize the map with the provided input values if exist
+  dashboard?.__inputs?.map((input) => {
+    grafanaInput[input.name] = input.value ?? '';
```

Comment:
Dumb question: why not call setInput here and do the escape in setInput ?

Reply:
calling setInput triggers a re-render, thus here it causes an infinite loop :/

Reply:
also we dont want the backslashes to appear in the text field, hence why afaiu there's no other choice that appending them right before the final submit